### PR TITLE
Prospects fixed

### PR DESCRIPTION
**Pull request by setayeshnri**
fixed the responsiveness of Prospects and added the gradient bg to how it works.

### CODE REVIEW

File: src/sections/HowItWorks/index.jsx [hunk 1]
```diff
@@ -5,15 +5,16 @@ const HowItWorks = () => {
...
       className="font-main items-center flex flex-col 
+
     "
     >
       <div>
         <SectionTitle>How it works</SectionTitle>
       </div>
-      <div className="w-full md:w-[700px] lg:w-[860px] xl:w-[1200px]">
-        <div className="w-full h-full  flex items-center justify-center ">
+      <div className="w-full flex justify-center bg-gradient-to-b p-4 from-transparent via-[#5F750B] to-transparent">
```
Comment (elen-oz) [inline src/sections/HowItWorks/index.jsx:14]: #5F750B - new color? we have all colours on tailwind.config file, let's use one of those

File: src/sections/HowItWorks/index.jsx [hunk 2]
```diff
@@ -5,15 +5,16 @@ const HowItWorks = () => {
...
+        <div className="w-full h-full  md:w-[700px] lg:w-[860px] xl:w-[1200px]  shadow-[0px_0px_0px_1px_#00000033,0px_2px_0px_0px_#00000014,0px_2px_6px_0px_#0000001a]">
```
Comment (elen-oz) [inline src/sections/HowItWorks/index.jsx:15]: new shadow for the video -  do we need it?

**Review by elen-oz — CHANGES_REQUESTED** [review]

File: src/sections/HowItWorks/index.jsx (hunk 2 above)
Comment (setayeshnri) [reply src/sections/HowItWorks/index.jsx:15]: it's in the design, we need it.

Comment (elen-oz) [reply src/sections/HowItWorks/index.jsx:15]: I don't see any shadow around video on design and I dont see any difference with and without that shadow on the website

Comment (setayeshnri) [reply src/sections/HowItWorks/index.jsx:15]: If you click on the video there are 3 drop shadows on it! they're subtle but they help with making the video pop out more.

**Review by elen-oz — APPROVED** [review]

**Review by Sheriff702 — APPROVED** [review]

**Pull request merged**